UI/src/view/HeightLimitLabel.js

```diff
@@ -0,0 +1,55 @@
+(function(root) {
+
+  root.HeightLimitLabel = function(){
+    AssetLabel.call(this, this.MIN_DISTANCE, this.populatedPoints);
+    var me = this;
+    var IMAGE_HEIGHT = 23;
+    this.MIN_DISTANCE = 5;
+    this.populatedPoints = [];
+
+    me.renderFeaturesByPointAssets = function(pointAssets, zoomLevel){
+      me.clearPoints();
+      return me.renderGroupedFeatures(pointAssets, zoomLevel, function(asset){
+        return me.getCoordinateForGrouping(asset);
+      });
+    };
+
+    var backgroundStyle = function (counter) {
+      return new ol.style.Style({
+        image: new ol.style.Icon(({
+          src: getImage(),
+          anchor : [0.5, 0.5 + (counter)]
```

Comment:
Considering the suggestion given on AssetLabel for the use of the index in here we should change the anchor for  `anchor : [0.5, 1.5 + (counter)]`